Realme/6/AndroidManifest.xml

```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8" standalone="no"?><manifest xmlns:android="http://schemas.android.com/apk/res/android" package="me.phh.treble.overlay.realme.6" platformBuildVersionCode="27" platformBuildVersionName="8.1.0">
+    <overlay android:isStatic="true" android:priority="96" android:requiredSystemPropertyName="ro.build.overlay.deviceid" android:requiredSystemPropertyValue="+RMX2001" android:targetPackage="android"/>
```

Comment:
split it on multi lines like the other overlays for a better readability

Reply:
okay sir